discovery-provider/src/tasks/audius_data.py

```diff
@@ -77,8 +77,7 @@ def audius_data_state_update(
                             txhash,
                         )
 
-                    if action == "Create":
-                        playlist_id = entity_id
+                    if action == "Create" or action == "Update":
```

Comment:
Generally would be nicer to create an enum and use here so it's never inconsistent - eventually I believe this will also be done for tracks and users

Reply:
Could be nice to add enum for entity_type too, but again not necessary for this